src/components/ViewTeams/container.js

```diff
@@ -22,12 +23,21 @@ export default Wrapped =>
         );
     };
 
+    selectTeam = (selectedTeam, clientToRefresh) => {
+      this.setState({ selectedTeam });
+      if (clientToRefresh) {
+        this.teamSearch(clientToRefresh);
+      }
+    };
+
     render() {
       return (
         <Wrapped
           {...this.props}
```

Comment:
Be more explicit than passing all props down. Only the history prop is being used by the child component.